### DISCUSSION TITLE
Hasura metadata is inconsistent with database on fresh local dev

### DISCUSSION THREAD

**Discussion by globetro** [nhost/nhost#1967] [Q&A]
I have a completely new project and just did a `nhost init --remote` followed by a `nhost dev`. It automatically opens the Hasura console webapp and immediately redirects me to the "Metadata status" page with an error: "You have been redirected because your GraphQL Engine metadata is in an inconsistent state".

Any idea why this happens and how do I fix it? This is with nhost CLI v0.9.7 for darwin-arm64. I've tried doing "Reload Metadata" along with "Reload all remote schemas" and "Reload all databases" and still doesn't work.

Here's a screenshot of the metadata error:
<img width="1290" alt="Screenshot 2023-05-25 at 9 07 10 PM" src="https://github.com/nhost/nhost/assets/9273/1bb6140a-b606-4c29-a611-fa36a5591fce">

This is what the auth tables look like (I'm not sure why the `refresh_token_types` table is a different icon):
<img width="1045" alt="Screenshot 2023-05-25 at 9 35 39 PM" src="https://github.com/nhost/nhost/assets/9273/07f31652-bfc4-43c8-8d42-812fd1009a4e">

And if I click on the `refresh_token_types` table, I get this error:
<img width="930" alt="Screenshot 2023-05-25 at 9 36 01 PM" src="https://github.com/nhost/nhost/assets/9273/a80cd3c8-fd1b-4193-a80d-5faa30bcc1fb">

**Comment by globetro** [comment]
I noticed that nhost cloud is running Hasura graphql-engine v2.25.1-ce while local development is running v2.15.2. Not sure if that's the cause of the issue. I couldn't find a way to get my local dev to use v2.25.1.

**Comment by dbarrosop** [comment]
The issue is because of the mismatch in hasura-auth, apologies for that. Could you do the following? Instead of using the nhost cli 0.9.7 use the latest [cli beta](https://github.com/nhost/cli/releases/tag/v1.0.0-beta3) and run:

```
nhost user login
nhost project link
nhost config pull
```

This should download an exact copy of the configuration of your cloud environment to use in local development. Docs for the new CLI are not 100% ready (that's why it is still in beta) but you can find some drafts [here](https://github.com/nhost/nhost.toml/tree/main/docs)

**Reply by globetro** [reply]
Thanks for the quick reply!

I tried it with the latest beta cli and it's working now (although minor thing, I had to do a fresh init, doing the link and pull still resulted in same issue).

An issue I'm seeing with the beta cli is when I visit the dashboard `https://local.dashboard.nhost.run`, I get this error in Chrome: `You cannot visit local.dashboard.nhost.run right now because the website uses HSTS. Network errors and attacks are usually temporary, so this page will probably work later.` Annoyingly, Chrome doesn't even give me the option to ignore the error, but luckily Safari does so I at least have a workaround.

**Reply by dbarrosop** [reply]
Would you mind redownloading beta3?

https://github.com/nhost/cli/releases/tag/v1.0.0-beta3

I pulled previous release and made a new one, turns out that the cert generation build failed and I missed it.

**Reply by globetro** [reply]
That seems to have fixed it, thanks!
